UnitTests/fixtures/LocStructuredOnly.strings

```diff
@@ -0,0 +1,3 @@
+"settings.navigation-bar.title.even.deeper" = "Settings";
+"SETTINGS.USER_PROFILE_SECTION.HEADER_TITLE" = "User Profile Settings";
+"seTTings.uSer-proFile-sectioN.footer_text" = "Here you can change some user profile settings.";
```

Comment:
Could be interesting to add some examples with parameters, to ensure that template properly handle Localized strings with placeholders. Like:

```
"settings.user_profile_section.points" = "Welcome %@! You have %d points on your account.";
```